### PR TITLE
Test: Update the Article Summary package to enable Jest Configurator

### PR DESCRIPTION
**Pull request by jimhunty**
As part of the Article Summary work, we wished to update the test suite Jest Configurator to make sure we are covering all platforms.

### CODE REVIEW

**Comment by colincclark** [comment]
This should merge straight into master, it is not dependent on the feature branch and does not help deliver 2 related articles

File: packages/article-summary/__tests__/android/article-summary.android.test.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import ArticleSummary from "../../article-summary";
+import test from "../article-summary-helper";
```
Comment (colincclark) [inline packages/article-summary/__tests__/android/article-summary.android.test.js]: can you rename this `shared` as per other packages? `test` is a Jest method already

Comment (jimhunty) [reply packages/article-summary/__tests__/android/article-summary.android.test.js]: Done

**Comment by jimhunty** [comment]
@colincclark We need to have your related articles work completed first so we can finish this refactor, otherwise it will break master. Your plan is to merge that into a feature branch so we have to follow suit.

**Pull request merged**